src/libraries/System.Speech/src/System.Speech.csproj

```diff
@@ -0,0 +1,256 @@
+    <Compile Include="Recognition\SrgsGrammar\SrgsRulesCollection.cs" />
+    <Compile Include="Recognition\SrgsGrammar\SrgsSemanticInterpretationTag.cs" />
+    <Compile Include="Recognition\SrgsGrammar\SrgsSubset.cs" />
+    <Compile Include="Recognition\SrgsGrammar\SrgsTagFormat.cs" />
+    <Compile Include="Recognition\SrgsGrammar\SrgsText.cs" />
+    <Compile Include="Recognition\SrgsGrammar\SrgsToken.cs" />
+    <Compile Include="Result\RecognitionEventArgs.cs" />
+    <Compile Include="Result\RecognitionResult.cs" />
+    <Compile Include="Result\RecognizedAudio.cs" />
+    <Compile Include="Result\RecognizedPhrase.cs" />
+    <Compile Include="Result\RecognizedWordUnit.cs" />
+    <Compile Include="Result\ReplacementText.cs" />
+    <Compile Include="Result\SemanticValue.cs" />
+    <Compile Include="Synthesis\BookmarkEventArgs.cs" />
+    <Compile Include="Synthesis\FilePrompt.cs" />
+    <Compile Include="Synthesis\InstalledVoice.cs" />
+    <Compile Include="Synthesis\PhonemeEventArgs.cs" />
+    <Compile Include="Synthesis\Prompt.cs" />
+    <Compile Include="Synthesis\PromptBuilder.cs" />
+    <Compile Include="Synthesis\PromptEventArgs.cs" />
+    <Compile Include="Synthesis\PromptStyle.cs" />
+    <Compile Include="Synthesis\SpeakCompletedEventArgs.cs" />
+    <Compile Include="Synthesis\SpeakProgressEventArgs.cs" />
+    <Compile Include="Synthesis\SpeechSynthesizer.cs" />
+    <Compile Include="Synthesis\SynthesizerStateChangedEventArgs.cs" />
+    <Compile Include="Synthesis\VisemeEventArgs.cs" />
+    <Compile Include="Synthesis\VoiceChangeEventArgs.cs" />
+    <Compile Include="Synthesis\VoiceInfo.cs" />
+    <Compile Include="Synthesis\TTSEngine\SAPIEngineTypes.cs" />
+    <Compile Include="Synthesis\TTSEngine\TTSEngineTypes.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="upstable_chs.upsmap">
+      <LogicalName>upstable_chs.upsmap</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="upstable_cht.upsmap">
+      <LogicalName>upstable_cht.upsmap</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="upstable_deu.upsmap">
+      <LogicalName>upstable_deu.upsmap</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="upstable_enu.upsmap">
+      <LogicalName>upstable_enu.upsmap</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="upstable_esp.upsmap">
+      <LogicalName>upstable_esp.upsmap</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="upstable_fra.upsmap">
+      <LogicalName>upstable_fra.upsmap</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="upstable_jpn.upsmap">
+      <LogicalName>upstable_jpn.upsmap</LogicalName>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <ProjectReference Include="$(LibrariesProjectRoot)Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj" />
```

Comment:
ProjectReferences shouldn't be conditioned per OS. (I just learnt this today: https://github.com/dotnet/runtime/issues/46338#issuecomment-749758264)